### PR TITLE
➕ cld-picture

### PR DESCRIPTION
**Pull request by arturkulig**

### CODE REVIEW

File: src/helpers/resolveMedia.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    return `${v}px`;
+  }
+  if (typeof v === "string") {
+    return v;
+  }
+  const valueOfValue = v.valueOf();
+  if (valueOfValue !== v) {
+    return resolveUnit(valueOfValue);
+  }
+  return resolveUnit(v.toString());
```
Comment (tocker) [inline src/helpers/resolveMedia.js]: When will this line be invoked?

File: src/components/CldPicture.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+See the [Image transformation reference](https://cloudinary.com/documentation/image_transformation_reference) documentation for all the options accepted by the `CldPicture` and `CldTransformation` components.
+
+### Events
+
+Use `v-on:*.native` to listen to native DOM events. `CldPicture` outputs an `picture`, `img` and `source`s elements and does not have any events on it's own.
+
+```jsx
+<CldPicture publicId="face_top" cloudName="demo"
+    :sources="[
```
[the remark this answers is not in the document]
Comment (arturkulig) [reply src/components/CldPicture.md]: - media attribute removed
- min_width, max_width added

min_width, max_width can be number and then the value is made into string with 'px' at the end.

File: src/components/CldPicture.vue [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+      return {
+        class: className,
+        attrs: merge(normalizeRest(this.$attrs), htmlAttrs)
+      };
+    },
+
+    imageOptions() {
+      const className = {
+        "cld-picture_image": true
```
[the remark this answers is not in the document]
Comment (arturkulig) [reply src/components/CldPicture.vue]: nowhere in SDK - it's just so users of the SDK can easily write selectors that reach that element

File: src/components/CldSource.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+Refer to the base [Cloudinary JS SDK](https://github.com/cloudinary/cloudinary_js#configuration) for configuration options.
+
+See the [Image transformation reference](https://cloudinary.com/documentation/image_transformation_reference) documentation for all the options accepted by the `CldPicture` and `CldTransformation` components.
+
+```jsx
+<cld-context cloudName="demo">
+    <picture>
+        <cld-source
+            publicId="face_top"
+            :media="{ screen: true, orientation: 'landscape', minHeight: 700 }"
```
[the remark this answers is not in the document]
Comment (arturkulig) [reply src/components/CldSource.md]: Complex media resolution was also not specified for this element and is now removed. `media` is now regular attribute.

File: src/helpers/resolveMedia.js (hunk 1 above)
Comment (arturkulig) [reply src/helpers/resolveMedia.js]: Complex media resolution was also not specified for this element and is now removed. This file no longer exists.

**Pull request closed without merging**